portal-api/src/v1/controllers/utilisation-report-service/due-reports.controller.js

```diff
@@ -0,0 +1,83 @@
+const api = require('../../api');
+const { getMonthName } = require('./previous-reports.controller');
+
+const getMostRecentReport = (reports) => {
+  if (!reports || reports.length === 0) {
```

Comment:
I have removed this method, but the idea was that I wanted to specifically check that the `reports` array is empty. I'd worry that `!reports.length` also implies that I am checking whether or not `length` exists on `reports`, but I know it always if `reports` exists because no other type can be returned by the `central-api` endpoint